leda/components/Collapse/Collapse.tsx

```diff
@@ -21,6 +22,8 @@ export const Collapse: CollapseComponent = (props: CollapseProps): React.ReactEl
 
   const theme = useTheme(themeProp, COMPONENTS_NAMESPACES.collapse);
 
+  // todo optimization (remove eslint disable to get further info)
+  // eslint-disable-next-line react/jsx-no-constructed-context-values
```

Comment:
I believe you could just wrap it in useMemo and it will do the thing

Reply:
A bit scary without tests